### PR TITLE
feat(replays): Add InvalidReplay outcome

### PR DESCRIPTION
**Pull request by cmanallen**

### CODE REVIEW

File: relay-server/src/actors/outcome.rs [hunk 1]
```diff
@@ -326,6 +326,9 @@ pub enum DiscardReason {
     /// (Relay) The profile is parseable but semantically invalid. This could happen if
     /// profiles lack sufficient samples.
     InvalidProfile,
+
+    // (Relay) We failed to parse the replay so we discard it.
+    InvalidReplay,
```
Comment (JoshFerge) [inline relay-server/src/actors/outcome.rs]: should we do `InvalidReplayEvent`? just to split ambiguity between event / segment

Comment (cmanallen) [reply relay-server/src/actors/outcome.rs]: 👍

**Review by JoshFerge — APPROVED** [review]
lgtm 👍🏼

**Review by olksdr — APPROVED** [review]
lgtm

**Pull request merged**